### DISCUSSION TITLE
On my Installation the Displaying Windows have un docked is there any way to dock them again

### DISCUSSION THREAD

**Discussion by ZafarAhmad74** [pawelsalawa/sqlitestudio#4189] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Move the window back over the left or right side of main window and while you're dragging it, the drop zone should highlight the area onto which you would drop it if you release the button.

**Reply by Gyrdleah** [reply]
In case its a problem for any one else, I could only get the 'sub windows' to dock if the main programme window was maximised.
